### PR TITLE
Updates naming convention for main RAT variables

### PR DESCRIPTION
**Pull request by DrPaulSharp**

### CODE REVIEW

**Review by StephenNneji — APPROVED** [review]
Please see comments

File: examples/bayesBenchmark/getFitValues.m [hunk 1]
```diff
@@ -1,11 +1,10 @@
 function [fitPars,fitNames,fitConstr] = getFitValues(inputProblem,controls)
 
-[problemDef,problemDef_cells,problemDef_limits,priors,controls] = RatParseClassToStructs_new(inputProblem,controls);
+[problemStruct,problemCells,problemLimits,~,controls] = parseClassToStructs(inputProblem,controls);
```
[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply examples/bayesBenchmark/getFitValues.m:3]: Yes, I largely left the examples as they are being replaced.

File: API/RAT.m [hunk 2]
```diff
@@ -1,44 +1,42 @@
-function [outProblemDef,result] = RAT(problemDefInput,controls)
+function [project,result] = RAT(project,inputControls)
```
[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply API/RAT.m:1]: I wanted to capture the fact that the controls are converted from a class object to a struct, but I'll take another look.

[the remark this answers is not in the document]
Comment (DrPaulSharp) [reply API/RAT.m]: I think the lesser evil here is to leave it as controls throughout, despite the change in type. What do you think?

**Pull request merged**